packages/realm_dart/lib/src/scheduler.dart

```diff
@@ -22,9 +24,11 @@ class Scheduler {
     _receivePort.handler = (dynamic message) {
       if (message is List) {
         // currently the only `message as List` is from the logger.
-        final level = message[0] as int;
-        final text = message[1] as String;
-        Realm.logger.log(LevelExt.fromInt(level), text);
+        final category = RealmLogCategory.fromString(message[0] as String);
+        // category is a string to avoid converting back and forth between RealmLogCategory and String
```

Comment:
This comment seems incorrect/unclear since `category` is in fact `RealmLogCategory`.

Reply:
old cruft, will fix.